components/use-cases/advertising-platforms/advertising-platforms.md

```diff
@@ -0,0 +1,215 @@
+
+Authorize your Google account and select the period for extracting the data.
+
+## Authorizing Data Destinations
+To create a working flow, you must select at least one data destination.
+
+### BigQuery Database
+
+![BigQuery Destination](/components/use-cases/advertising-platforms/bigquery-destination.png)
+
+To configure the destination component, you need to set up a [Google Service Account](https://console.cloud.google.com/iam-admin/serviceaccounts) and create a new JSON key.
+
+A detailed guide is available [here](https://help.keboola.com/components/writers/database/bigquery/).
+
+### Google Sheets
+
+![Google Sheets Destination](/components/use-cases/advertising-platforms/google-sheets-destination.png)
+
+Authorize your Google account.
+
+Duplicate the sheet into your Google Drive and paste the file ID back to Keboola Connection. It is needed for correct mapping 
+in your duplicated Google sheet. 
+
+*Note: We are working on automatization. In the future, you won't have to duplicate the sheet by yourself, we will do that for you.*
+
+### Snowflake Database Provided by Keboola
+
+If you do not have your own data warehouse, follow the instructions and we will create a database for you: 
+
+1. After clicking **Save**, the template will be used in your project. You will see a flow. 
+2. Go there and click on **Snowflake Data Destination**. You will be redirected to the data destination configuration and asked to set up the database. 
+3. Select **Keboola Snowflake database**. 
+4. Then go back to the flow and click **Run**. 
+
+![DWH Provided by Keboola](/components/use-cases/advertising-platforms/dwh-provided-by-keboola.png)
+
+Everything is set up.
+
+### Snowflake Database
+
+If you want to use your own Snowflake database, you must provide the host name (account name), user name, password, database name, 
+schema, and a [warehouse](https://docs.snowflake.net/manuals/user-guide/warehouses.html).
+
+![Snowflake Destination](/components/use-cases/advertising-platforms/snowflake-destination.png)
+
+We highly recommend that you create a dedicated user for the destination component in your Snowflake database. Then you must provide 
+the user with access to the Snowflake [Warehouse](https://docs.snowflake.net/manuals/user-guide/warehouses.html). 
+
+**Warning:** Keep in mind that Snowflake is **case sensitive** and if identifiers are not quoted, they are converted to upper case. 
+So if you run, for example,  a query CREATE SCHEMA john.doe;, you must enter the schema name as DOE in the destination component configuration.
+
+More info [here](https://help.keboola.com/components/writers/database/snowflake/).
+
+## Most Common Errors
+Before turning to the Keboola support team for help, make sure your error is not a common problem that can be solved without our help.
+
+### Missing Credentials to Snowflake Database 
+If you see the error pictured below, you have probably forgotten to set up the Snowflake database. 
```

Comment:
btw, není to už fixnutý tohle?

Reply:
@MonikaFeigler

Reply:
máme to pořád v dokumentaci snowflake DB, takže pokud to fixnuté je, do dokumentace samotné se to ještě nedostalo.